_includes/manuals/1.8/4.3.5.3_ms_dhcp.md

```diff
@@ -155,22 +44,46 @@ _Sample config/settings.yml file_
  # valid options are
  # Logger::WARN, Logger::DEBUG, Logger::Error, Logger::Fatal, Logger:INFO, LOGGER::UNKNOWN
  #:log_level: Logger::DEBUG
+ :log_level: Logger::Error
+</pre>
+
+11. Edit config/settings.d/dhcp.yml so that it looks a bit like this
+
+_Sample config/settings.d/dhcp.yml file_
+<pre>
+ ---
+ # Can be true, false, or http/https to enable just one of the protocols
+ :enabled: true
+ # Enable DHCP management
+ :dhcp: true
+ # The vendor can be either isc or native_ms
+ :dhcp_vendor: native_ms
+ # The dhcp_server is only used by the native_ms implementation
+ :dhcp_server: 10.10.10.1
 </pre>
 
-10) Create the SSL key
+12. Create the SSL key
```

Comment:
You might have to use `12)` here as Markdown tries to restart the numbering from 1